electron/src/electron-main.js

```diff
@@ -174,7 +165,12 @@ function createWindow(url) {
     // in an array if your app supports multi windows, this is the time
     // when you should delete the corresponding element.
     win = null;
-});
+  });
+
+  win.webContents.on('will-navigate', function(e, url) {
```

Comment:
This doesnt affect the other wallet links?

Reply:
No, it doesn't affect the others.